tests/karma/BookNavigator/search/search-provider.test.js

```diff
@@ -23,21 +23,6 @@ describe('Search Provider', () => {
     expect(provider.menuDetails).to.exist;
     expect(provider.component).to.exist;
   });
-  describe('BR calls', () => {
-    it('`advanceToPage', () => {
```

Comment:
q: why do we no longer need this? was it a dupe of another fn?

Reply:
The logic moved to `_searchPluginGoToResult`; this class no longer needs to know how to advance pages, it just calls that fn directly.